File: pc010/rand.php
<?php

$maxRounds = 1000;
$i = 0;
$donewr = true;

do {
    if ($donewr) $rand = random_int(1,100);
    echo("Enter a number between 1 and 100, or enter 0 to quit: ");
    $entry = intval(trim(fgets(STDIN)));
    if ($entry == 0) exit(0);
    $donewr = false;
    if      ($entry == $rand)  { $donewr = true;
			       echo("You guessed it!  Let's play again!\n"); }
    else if ($entry <  $rand)  echo("The secret number is greater than $entry.  Please try again...\n");
    else if ($entry >  $rand)  echo("The secret number is less than $entry.  Please try again...\n");
    
} while(++$i <= $maxRounds);

echo("The game has been limited to $maxRounds rounds as a check against infinite loops.\n");

File: pc010/py/rand.py
# This is not my work.  I am running this to compare my version in PHP to this.

from random import randint
#range 1-100
def guessing_game():
    random_number = randint(1, 100)
    #get input guess from user.
    while True:
        guess = int(input('Guess the number between 1-100 or enter 0 to quit:'))
        if guess == 0:
            print('See ya!')
            break
        elif guess == random_number:
            print('You win! Let\'s play again :)')
            random_number = randint(1, 100)
        elif guess < random_number:
            print('Too low! Try again!')
        elif guess > random_number:
            print('Too high! Try again!')


guessing_game()
